pkgs/by-name/ne/neohtop/package.nix

```
@@ -0,0 +1,78 @@
+{
+  rustPlatform,
+  cargo-tauri,
+  nodejs,
+  jq,
+  stdenv,
+  fetchNpmDeps,
+  zsh,
+  fetchFromGitHub,
+  npmHooks,
+  lib,
+  openssl,
+}:
+
+let
+  tauri = cargo-tauri;
+
+in
+rustPlatform.buildRustPackage rec {
+  name = "neohtop";
+  pname = name;
+  version = "1.1.0";
+
+  src = fetchFromGitHub {
+    owner = "abdenasser";
+    repo = "neohtop";
+    rev = "v${version}";
+    hash = "sha256-23r/oBR4zWmeSVa5QaJ2wJOwzwmqYVhMKOpaEABP7Qs=";
+  };
+  buildInputs = [
+    openssl
+  ];
+  nativeBuildInputs = [
+    tauri.hook
+    npmConfigHook
+    nodejs
+  ];
+  cargoRoot = "src-tauri";
+  cargoLock.lockFile = ./Cargo.lock;
+  npmDeps = fetchNpmDeps {
+    inherit src;
+    hash = "sha256-6FWowWVVeU7U5aM4WGVWv0WmauLv/F4mtSdjNeeF6eQ=";
+  };
+  postPatch = ''
+    rm ${cargoRoot}/Cargo.lock
+    cp ${./Cargo.lock} ${cargoRoot}/Cargo.lock
+    chmod 755 ${cargoRoot}/Cargo.lock
+  '';
+  # remove macOS signing
+  postConfigure = ''
+    ${lib.getExe jq} 'del(.bundle.macOS.signingIdentity, .bundle.macOS.hardenedRuntime)' src-tauri/tauri.conf.json > tmp.json
```

Add jq as a nativeBuildInput and call it directly. Also, I *think* jq can write directly in place (I might be wrong)
```suggestion
    jq 'del(.bundle.macOS.signingIdentity, .bundle.macOS.hardenedRuntime)' src-tauri/tauri.conf.json > src-tauri/tauri.conf.json
```

---

unfortunatly jq doesnt have `-i` (or any other inplace functionality i know of)